helm/dns-operator-azure/templates/psp.yaml

```diff
@@ -33,4 +33,4 @@ spec:
   volumes:
     - "secret"
     - "configMap"
-{{- end }}
+{{- end }}
```

Comment:
Please add new line at the end of the file.

Reply:
fixed